langlib/lang.xml/src/main/java/org/ballerinalang/langlib/xml/CreateText.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *   Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.langlib.xml;
+
+import org.ballerinalang.jvm.XMLFactory;
+import org.ballerinalang.jvm.api.values.BString;
+import org.ballerinalang.jvm.values.XMLValue;
+
+/**
+ * Create XML text value from string.
+ *
+ * @since Swan Lake
+ */
+//@BallerinaFunction(
+//        orgName = "ballerina", packageName = "lang.xml", version = XML_VERSION,
+//        functionName = "createText",
+//        args = {@Argument(name = "string", type = TypeKind.STRING)},
+//        returnType = {@ReturnType(type = TypeKind.XML)},
+//        isPublic = true
+//)
```

Comment:
Shall we remove this? I don't think we'll need this again, even if we do, we can add it then.